core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/GraalVM.java

```diff
@@ -52,8 +54,8 @@ static Version parse(List<String> lines) {
             Matcher secondMatcher = SECOND_PATTERN.matcher(lines.get(1));
             Matcher thirdMatcher = THIRD_PATTERN.matcher(lines.get(2));
             if (firstMatcher.find() && secondMatcher.find() && thirdMatcher.find()) {
-                String javaVersion = firstMatcher.group(VNUM_GROUP);
-                java.lang.Runtime.Version v = null;
+                String javaVersion = secondMatcher.group(BUILD_INFO_GROUP);
+                java.lang.Runtime.Version v;
```

Comment:
This puzzled me for a while. Especially why you've picked BUILD_INFO_GROUP over  VNUM_GROUP. But it looks like the former is richer in terms of optional and build info. OK.

Reply:
Correct.